Resolve the compile warning of openmldb-batch

Now we can get logs of warning of Java module `openmldb-batch` when compiling.

```
[WARNING] /Users/tobe/code/4pd/OpenMLDB/java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/SparkPlanner.scala:342: warning: object Jav
aConversions in package collection is deprecated (since 2.12.0): use JavaConverters
[WARNING]       engine = new SqlEngine(sql, dbs, engineOptions, config.defaultDb)
[WARNING]                                   ^
```

You can reproduce the issue by compiling java modules with `mvn clean package`.

--- a/java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/SparkPlanner.scala
+++ b/java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/SparkPlanner.scala
@@ -35,7 +35,7 @@ import org.apache.spark.SparkFiles
 import org.apache.spark.sql.{DataFrame, SparkSession}
 import org.slf4j.LoggerFactory
 
-import scala.collection.JavaConversions.seqAsJavaList
+import scala.collection.JavaConverters.seqAsJavaList
 import scala.collection.mutable
 import scala.reflect.io.File
 
@@ -337,7 +337,7 @@ class SparkPlanner(session: SparkSession, config: OpenmldbBatchConfig, sparkAppN
     }
 
     try {
-      sqlEngine = new SqlEngine(dbs, engineOptions)
+      sqlEngine = new SqlEngine(seqAsJavaList(dbs), engineOptions)
       val engine = sqlEngine.getEngine
 
       // TODO(tobe): If use SparkPlanner instead of OpenmldbSession, these will be null

--- a/java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/utils/DataTypeUtil.scala
+++ b/java/openmldb-batch/src/main/scala/com/_4paradigm/openmldb/batch/utils/DataTypeUtil.scala
@@ -114,7 +114,6 @@ object DataTypeUtil {
       case com._4paradigm.hybridse.node.DataType.kDouble => DoubleType
       case com._4paradigm.hybridse.node.DataType.kBool => BooleanType
       case com._4paradigm.hybridse.node.DataType.kVarchar => StringType
-      case com._4paradigm.hybridse.node.DataType.kVarchar => StringType
       case com._4paradigm.hybridse.node.DataType.kDate => DateType
       case com._4paradigm.hybridse.node.DataType.kTimestamp => TimestampType
       case _ => throw new IllegalArgumentException(

